code/game/gamemodes/gang/dominator.dm

```diff
@@ -53,7 +53,7 @@
 			if(!warned && (time_remaining < 180))
 				warned = 1
 				var/area/domloc = get_area(loc)
-				gang.message_gangtools("Less than 3 minutes remain in hostile takeover. Defend your dominator at [domloc.map_name]!")
+				gang.message_gangtools("Less than 3 minutes remains in hostile takeover. Defend your dominator at [domloc.map_name]!")
```

Comment:
The change is contentious and looks wrong both ways; this should be reworded entirely.